domain/src/main/java/mihon/domain/extensionrepo/interactor/UpdateExtensionRepo.kt

```diff
@@ -0,0 +1,25 @@
+package mihon.domain.extensionrepo.interactor
+
+import eu.kanade.tachiyomi.network.NetworkHelper
+import mihon.domain.extensionrepo.api.ExtensionRepoApi
+import mihon.domain.extensionrepo.model.ExtensionRepo
+import mihon.domain.extensionrepo.repository.ExtensionRepoRepository
+
+class UpdateExtensionRepo(
+    private val extensionRepoRepository: ExtensionRepoRepository,
+    networkService: NetworkHelper,
+) {
+
+    private val extensionRepoApi = ExtensionRepoApi(networkService.client)
+
+    suspend fun await(repo: ExtensionRepo): ExtensionRepo? {
```

Comment:
where is the return used 🤔

Reply:
Good catch, was from an earlier revision.